ingest/author_de.py

```diff
@@ -220,28 +311,42 @@ def execute(self):
         metrics = []
         file_path = self.author_de_file
 
-        data = pd.read_csv(file_path)
+        # TODO:
+        #   - Throw well-formatted error if file type not in ALLOWED_FILE_TYPES
+        #   - Consider if / how this merges with centralized handling in ingest_files.py
+        file_type = IngestFiles.get_file_type(file_path)[0]
+        if file_type == 'text/tab-separated-values':
```

Comment:
Out of scope: I know we've had issues generally with ingest and sniffing delimiters.  In the Rails codebase, we're able to read & split lines on multiple delimiters by passing a regex like `/[,\t]/`.  Does `read_csv` have similar support for this?

Reply:
> Does read_csv have similar support for this?

It does!  From [`pandas.read_csv` docs](https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html), emphasis mine:


> ***sep str, default ‘,’***
Character or regex pattern to treat as the delimiter. If sep=None, the C engine cannot automatically detect the separator, but the Python parsing engine can, meaning the latter will be used and automatically detect the separator from only the first valid row of the file by Python’s builtin sniffer tool, csv.Sniffer. In addition, **separators longer than 1 character and different from '\s+' will be interpreted as regular expressions** and will also force the use of the Python parsing engine. Note that regex delimiters are prone to ignoring quoted data. Regex example: '\r\t'.
>
> ***delimiter str, optional***
Alias for sep.

I'll give that a try in my next PR.